sdk/loadtestservice/load-testing-rest/tsconfig.json

```diff
@@ -9,7 +9,10 @@
       "@azure-rest/load-testing": ["./src/index"]
     },
     "types": [],
-    "lib": ["DOM"]
+    "lib": ["DOM"],
+    "target": "ES6",
+    "module": "commonjs",
+    "skipLibCheck": true
```

Comment:
Why are these needed?

Reply:
These are needed to build project and run tests.

Reply:
They're non-standard, I would recommend a [standard one](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/cognitivelanguage/ai-language-text/tsconfig.json). I will mark this as unresolved for now.

Reply:
Will check and update this.